src/bitcoin-tx.cpp

```diff
@@ -854,7 +855,7 @@ static int CommandLineRawTx(int argc, char* argv[])
     return nRet;
 }
 
-int main(int argc, char* argv[])
+MAIN_FUNCTION
```

Comment:
I like this macro approach more but would it make sense to have the macro cover only contain  `__declspec(dllexport)` and the empty string? 

Hiding the function arguments obscures the code somewhat (though in that case, most people can probably infer `int argc, char* argv[]`).

Reply:
No opinion on the scope of the macro, both arguments make sense to me. It has no effect on detection.